cfgov/unprocessed/apps/find-a-housing-counselor/js/common.js

```diff
@@ -59,15 +59,21 @@ function scriptLoaded( evt ) {
  * Set access map options and create map.
  */
 function initializeMap() {
-  const fcm = document.querySelector( '#hud_search_container' );
-  fcm.classList.remove( 'no-js' );
-  window.L.mapbox.accessToken = mapboxAccessToken;
-  map = window.L.mapbox.map( 'hud_hca_api_map_container' )
-    .setView( [ 40, -80 ], 2 )
-    .addLayer( window.L.mapbox.styleLayer( 'mapbox://styles/mapbox/streets-v11' ) );
-
-  if ( hudData.counseling_agencies ) {
-    updateMap( hudData );
+  const showMap = Boolean(
+    document.getElementById( 'hud_hca_api_map_container' )
```

Comment:
Superbly trivial but `querySelector` is used everywhere else in this file to select IDs so consider using it here for consistency's sake.